### DISCUSSION TITLE
3代可以用1.2.0以下的penmods吗

### DISCUSSION THREAD

**Discussion by PetricWu** [PenUniverse/PenMods#224] [你问我答]
3代可以用1.2.0以下的penmods吗

**Comment by Redbeanw44602** [comment]
仅1.1.8支持G3

**Reply by PetricWu** [reply]
请问x3s 2.7.1的系统支持那个版本的penmods呢 
3Q

**Reply by Redbeanw44602** [reply]
> 请问x3s 2.7.1的系统支持那个版本的penmods呢 
> 3Q

暂时不支持3代

**Reply by PetricWu** [reply]
ok ：）

**Reply by PetricWu** [reply]
请问有希望支持3代吗

**Comment by laoaideqi** [comment]
> 仅1.1.8支持G3

我G3可以用社区里面的那个1.2.0安装程序安装 只是字体那些的变小了

**Reply by Redbeanw44602** [reply]
不建议强装..........打算到时候给三代出独立的方案
